docs/python_modules/enchantfilters/SWFFilters.py

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright (C) 2018 Pelagicore AB
+#
+# Permission to use, copy, modify, and/or distribute this software for
+# any purpose with or without fee is hereby granted, provided that the
+# above copyright notice and this permission notice appear in all copies.
+#
+# THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL
+# WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR
+# BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES
+# OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS,
+# WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION,
+# ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS
+# SOFTWARE.
+#
+# For further information see LICENSE
+
+from enchant.tokenize import Filter
+import string
+
+def all_match(string, allowed_chars):
+    return all(c in allowed_chars for c in string)
+
+class GitCommitFilter(Filter):
+    """ If a word looks like a git commit hash (long or short), ignore it """
+
+    def _skip(self, word):
+        allowedchars = string.hexdigits.lower()
```

Comment:
This should be a class var since it's always the same.

Reply:
Good point!